packages/vite/src/node/http.ts

```diff
@@ -129,18 +129,18 @@ export async function resolveHttpsConfig(
 
   const { ca, cert, key, pfx } = httpsOption
   Object.assign(httpsOption, {
-    ca: readFileIfExists(ca),
-    cert: readFileIfExists(cert),
-    key: readFileIfExists(key),
-    pfx: readFileIfExists(pfx),
+    ca: await readFileIfExists(ca),
```

Comment:
Promise.all ??

Reply:
Good idea, done at [vitejs/vite@`de5e611` (#12553)](https://github.com/vitejs/vite/pull/12553/commits/de5e61134a47c7347aaa883726504a0b3367d2fa). Also simplified the logic.